tools/azure-sdk-tools/packaging_tools/code_report.py

```diff
@@ -149,22 +149,6 @@ def merge_report(report_paths):
         merged_report["operations"].update(report_json["operations"])
     return merged_report
 
-def filter_track2_versions(package_name, versions):
-    track2_versions = {
-        'azure-mgmt-appconfiguration': '1.0.0',
-        'azure-mgmt-compute':'17.0.0',
-        'azure-mgmt-eventhub':'8.0.0',
-        'azure-mgmt-keyvault':'7.0.0',
-        'azure-mgmt-monitor':'1.0.0',
-        'azure-mgmt-network':'16.0.0',
-        'azure-mgmt-resource':'15.0.0',
-        'azure-mgmt-storage':'16.0.0',
-    }
-    upbound = track2_versions.get(package_name)
-    if not upbound:
-        return versions
-    return list(filter(lambda x: x < upbound, versions))
```

Comment:
Why remove this, which is added before

Reply:
The filter_track2_versions() is for track1 code_report.
And currently we are using release/v3 branch for track1. 
This PR remove it from master.